train_DTI.py

```diff
@@ -41,9 +41,9 @@
 
 parser = ArgumentParser(description="PLM_DTI Training.")
 parser.add_argument(
-    "--exp-id", required=True, help="Experiment ID", dest="experiment_id"
+    "--exp-id", help="Experiment ID", dest="experiment_id", default="tmp"
```

Comment:
Okay to make log file default, but would prefer to have experiment ID explicitly set. That way default behavior is not to possibly overwrite previously existing runs

Reply:
I reverted this change.
The motivation was to allow running the code as a script for debugging purposes. when "required" is True it's not possible and you have to run it as `python train_DTI.py` and supply CLI arguments.
I understand there are pros and cons to each method and there are also ways to debug with CLI arguments, no big deal.